platform/el/kanidm.spec

```diff
@@ -0,0 +1,186 @@
+Name:           kanidm
```

Comment:
We don't carry spec files upstream - these are probably better in something like https://build.opensuse.org/project/show/network:idm where we actually would build and publish these.